nameko/cli/shell.py

```diff
@@ -15,7 +15,7 @@
 
 def init_parser(parser):
     parser.add_argument(
-        '--broker', default='amqp://guest:guest@localhost:5672/nameko',
+        '--broker', default='amqp://guest:guest@localhost:5672/',
```

Comment:
single slash on purpose? (which i think makes vhost `None` meaning use amqp lib defaults)

we could use more defaults by having our default be just `amqp://localhost`